numpy/ma/core.py

```diff
@@ -26,11 +26,7 @@
 import textwrap
 import re
 from functools import reduce
-
-if sys.version_info[0] >= 3:
-    import builtins
-else:
-    import __builtin__ as builtins
+import builtins
```

Comment:
The use of builtins here is to make sure we are not using `np.all`, `np.max` ... Perhaps it can be avoided.

Reply:
if we're keeping this import, group it up to L23